sklearnex/preview/decomposition/pca.py

```diff
@@ -28,6 +28,8 @@
 from sklearn.utils.validation import check_array
 from sklearn.base import BaseEstimator
 from sklearn.utils.validation import check_is_fitted
+if sklearn_check_version('1.1') and not sklearn_check_version('1.2'):
+    from sklearn.utils import check_scalar

```

Comment:
Why
 these changes are in Random Forest related PR?

Reply:
Too small for separate PR